src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -79,225 +123,455 @@ const useStyles = makeStyles()(() => ({
+                            <InputLabel id="frequency-label">
+                              {t('Frequency')}
+                            </InputLabel>
+                            <Select
+                              className={classes.select}
+                              inputProps={{
+                                'data-testid': 'pledgeFrequency-input',
+                              }}
+                              data-testid="pledgeFrequency"
+                              label={t('Frequency')}
+                              labelId="frequency-label"
+                              placeholder="Frequency"
+                              style={{ width: '100%' }}
+                              value={pledgeFrequency}
+                              onChange={(event) =>
+                                setFieldValue(
+                                  'pledgeFrequency',
+                                  event.target.value,
+                                )
+                              }
+                            >
+                              <MenuItem value="Frequency" disabled>
+                                {t('Frequency')}
+                              </MenuItem>
+                              {Object.entries(frequencies).map(
+                                ([freqValue, freqTranslated]) => (
+                                  <MenuItem
+                                    value={freqValue}
+                                    key={freqValue}
+                                    data-testid="pledgeFrequencyOptions"
+                                  >
+                                    {freqTranslated}
+                                  </MenuItem>
+                                ),
+                              )}
+                            </Select>
+                            <FormHelperText
+                              error={true}
+                              data-testid="pledgeFrequencyError"
+                            >
+                              {errors.pledgeFrequency && errors.pledgeFrequency}
+                            </FormHelperText>
+                          </Box>
+                        </Grid>
+                      </Grid>
+                    </Grid>
+                    {!!donations.length && (
+                      <Grid
+                        container
+                        className={classes.donationsTable}
+                        lg={12}
+                      >
+                        {donations.map((donation) => (
+                          <Grid
+                            key={donation.amount.conversionDate}
+                            display="flex"
+                            flexDirection="column"
+                          >
+                            <Box>
+                              <Typography
+                                fontWeight={700}
+                                data-testid="donationDate"
+                              >
```

Comment:
```suggestion
                              <Typography
                                fontWeight={700}
                                variant="body2"
                                data-testid="donationDate"
                              >
```

Reply:
I would try making all this text smaller. Like the body2 variant. Then maybe you could have it show the past 6 donations instead of just 5.